ted_sws/rdf_differ/services/difference_between_rdf_files.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+import rdflib
+from rdflib import Graph
+from rdflib.compare import isomorphic, graph_diff
+
+from tests import TEST_DATA_PATH
+
+FIRST_RML_FILE = TEST_DATA_PATH / "technical_mapping_F03.rml.ttl"
+SECOND_RML_FILE = TEST_DATA_PATH / "technical_mapping_F06.rml.ttl"
+
+
+def rdf_differ_service(FIRST_RML_FILE: str, SECOND_RML_FILE: str):
+    """
+    Given two RML files representing turtle-encoded RDF,
+    check whether they represent the same graph.
+    """
+    first_grath = Graph().parse(FIRST_RML_FILE, format='turtle')
+    second_grath = Graph().parse(SECOND_RML_FILE, format='turtle')
+    eq = isomorphic(first_grath, second_grath)
```

Comment:
Use vars name more semantically, instead of "eq" use "is_equal_graphs" or "check_graph_equality".